clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3095,6 +3095,21 @@ static bool checkWorkGroupSizeValues(Sema &S, Decl *D, const ParsedAttr &AL) {
   return Result;
 }
 
+static Expr *checkWorkSizeAttrExpr(Sema &S, const ParsedAttr &CI, Expr *E) {
```

Comment:
Oh! Should the other function with the same name be removed from Sema.h?

Reply:
I have updated patch that no longer use new function checkWorkSizeAttrExpr() that i added in previous patch anymore.